docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1699 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    pandas_upper = timeit.timeit(lambda : pd_series.str.upper(), number=20)
</pre><p>Can we write all these benchmarks in a more reusable style? Maybe track the results in a dictionary?</p><pre class="ql-syntax" spellcheck="false">def bench(pdf, gdf, func, **kwargs):
    pdf_time = timeit.timeit(func(pdf), **kwargs)
    gdf_time = timeit.timeit(func(gdf), **kwargs)
    return pdf_time, gdf_time

upper = bench(pd_series, gd_series, lambda df: df.str.upper(), number=20)
contains = bench(pd_series, gd_series, lambda df: df.str.contains(r"[0-9][a-z]"), number=20)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
Done 👍